public/javascripts/SVLabel/src/SVLabel/data/Form.js

```diff
@@ -16,9 +16,10 @@
  */
 function Form (labelContainer, missionModel, missionContainer, navigationModel, neighborhoodModel, panoramaContainer, taskContainer, mapService, compass, tracker, params) {
     var self = this;
-    var properties = {
+    let properties = {
         dataStoreUrl : undefined,
-        beaconDataStoreUrl : undefined
+        beaconDataStoreUrl : undefined,
+        timeLastQuried : new Date().getTime() // Assuming that Form is created when TaskContainer is created
```

Comment:
I would change the comment to say something like "The priorities are up-to-date when the page loads". Again, would rename to something like `lastStreetPriorityUpdate`.